src/vmm/src/arch/aarch64/gic/regs.rs

```diff
@@ -80,7 +52,7 @@ pub(crate) trait MmioReg {
 
 pub(crate) trait VgicRegEngine {
     type Reg: MmioReg;
-    type RegChunk: Clone + Default + Versionize;
+    type RegChunk: Clone + Default + Serialize + for<'a> Deserialize<'a>;
```

Comment:
silly question probably, but is it passing on `aarch64`?

Reply:
Yeah, `cargo check --aarch64-unknown-linux-gnu` passes. We also dont implement serialize and deserialize for any of the types that implement the trait. I think it works because we dont ever use this trait in a generic fashion during (de)serialization, and all concrete types for which it is implemented provide a `RegChunk` type that is serializable.

Reply:
done!